src/client/components/HomeDescription/HomeDescription.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './HomeDescription.styles.css';
+
+export default function HomeDescription() {
+  return (
+    <div className="home-description">
+      <p className="home-description-upper">
+        <b>GuideIT </b>
```

Comment:
Please use `strong` instead of `b`. Nowadays we prefer `em` instead of `i` and `strong` instead of `b`.

Reply:
New info for me. I will do that, Thank you for your review